src/components/Button/Button.module.scss

```diff
@@ -91,21 +92,24 @@ $disabled-on-primary-color-opacity: 0.5;
 }
 
 .sizeXxs {
-  @include font-input();
+  @include smoothing-text;
```

Comment:
Why don't we include it in `vibe-text`? We don't always need it?

Reply:
I know it's looking exactly like before, but the font doesn't seem smooth to me here... Am I wrong?
![image](https://github.com/mondaycom/monday-ui-react-core/assets/104433616/b43a3589-1a51-46f7-9b0e-e63b2db14a6e)

Reply:
Meirav decision (regarding smoothing text).
Regarding text issue - will double check and update here

Reply:
look the same to me